crates/biome_deserialize/src/visitor.rs

```diff
@@ -3,34 +3,26 @@ use biome_rowan::{Language, SyntaxNode};
 
 /// Generic trait to implement when resolving the configuration from a generic language
 pub trait VisitNode<L: Language>: Sized {
-    /// Called when visiting the key of a member
-    fn visit_member_name(
+    /// Called when visiting a plain value, i.e. a value that is not a list.
+    fn visit_plain_value(
```

Comment:
`visit_value`, and the documentation should explicitly list the values that aren't/are visited.